### DISCUSSION TITLE
after install interactsh I get an error

### DISCUSSION THREAD

**Discussion by the-lilcrazy** [projectdiscovery/interactsh#763] [General]
![Capture](https://github.com/projectdiscovery/interactsh/assets/121715952/9e06993a-190a-4bc0-b53c-2a63400f1161)
I try to change DNS in /etc/resolv.conf 
and I check the network!

**Comment by olearycrew** [comment]
Hi @the-lilcrazy 

Can you provide the output of these commands?

```
interactsh-client -hc
```

```
dig oast.fun
dig oast.site
dig oast.me
```
